src/Uno.UI.Dispatching/DispatchingFeatureConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Uno;
+
+public static class DispatchingFeatureConfiguration
+{
+	internal static class DispatcherQueue
+	{
+		public static TimeSpan FrameDuration { get; set; } = TimeSpan.FromMilliseconds(1 / 60.0);
```

Comment:
Does 120 FPS need to be configured manually?

Maybe we could try to determine the FPS based on the system at runtime? https://developer.android.com/media/optimize/performance/frame-rate

For example Android and iOS drop display refresh rate when low battery mode is on, I guess it should be possible to get notified about that fact too

Reply:
This is specific to wasm rendering, I'll update it.